### PR TITLE
IA-796: Adding RDS to AWS Backup Service

### PR DESCRIPTION
**Pull request by bushong1**

### CODE REVIEW

File: terraform/base/modules/ckan/aws-backup.tf [hunk 1]
```diff
@@ -25,18 +25,59 @@ data "aws_iam_policy_document" "efs_backup" {
...
+    resources   = [
+      "arn:aws:rds:${var.region}:${var.account_id}:db:${terraform.workspace}-ckan",
+      "arn:aws:rds:${var.region}:${var.account_id}:snapshot:${terraform.workspace}-ckan-*",
+    ]
+  }
+  statement {
+    sid         = "AWSBackupManageSnapshots"
+    actions     = [ "rds:DeleteDBSnapshot" ]
+    resources   = [
+      "arn:aws:rds:*:*:snapshot:awsbackup:*"
```
Comment (elijahmuha) [inline terraform/base/modules/ckan/aws-backup.tf]: Any reason this is as broad as it is?

File: terraform/base/modules/ckan/aws-backup.tf [hunk 2]
```diff
@@ -79,6 +123,7 @@ resource "aws_backup_selection" "efs_weekly" {
 
   resources = [
     aws_efs_file_system.efs[0].arn,
+    aws_db_instance.ckan_rds[0].arn,
```
Comment (elijahmuha) [inline terraform/base/modules/ckan/aws-backup.tf]: Maybe nitpicky but I don't like that this is included in a resourced named `efs_weekly` when it isn't efs.

File: terraform/base/modules/ckan/aws-backup.tf [hunk 3]
```diff
@@ -90,5 +135,6 @@ resource "aws_backup_selection" "efs_quarterly" {
...
```
Comment (elijahmuha) [inline terraform/base/modules/ckan/aws-backup.tf]: Same as idea as above

File: terraform/base/modules/ckan/aws-backup.tf (hunk 1 above)
Comment (bushong1) [reply terraform/base/modules/ckan/aws-backup.tf]: As far as I understand it, this is a service role for the AWS Backups service, and the snapshots created by them are in the "awsbackup" namespace.

File: terraform/base/modules/ckan/aws-backup.tf (hunk 2 above)
Comment (bushong1) [reply terraform/base/modules/ckan/aws-backup.tf]: yup. gotta fix that.

**Pull request merged**